shared/utility/support/Timer.hpp

```diff
@@ -1,30 +0,0 @@
-#ifndef TIMER_HPP
-#define TIMER_HPP
-
-#include <chrono>
-
-class Timer {
```

Comment:
I would keep the timer, It will never really stay in code, but you use it when you want to time a section of code (and then don't commit your performance prints when you finish)

Reply:
Fair. Will remove this change

Reply:
I say this because you have no idea how many times I've written a timer module and just hacked it and wished I had something easier to use :P